src/appleseed-max-impl/appleseedrenderer/projectbuilder.cpp

```diff
@@ -990,7 +990,8 @@ namespace
         const bool has_lights = !entities.m_lights.empty();
         const bool has_emitting_mats = has_light_emitting_materials(material_map);
         const bool has_emitting_env = !scene.get_environment()->get_parameters().get_optional<std::string>("environment_edf").empty();
-        if (!has_lights &&
+        if (rend_params.inMtlEdit ||
```

Comment:
Let's add parentheses to make the logic clearer:

    if (rend_params.inMtlEdit ||
        (!has_lights &&
         !has_emitting_mats &&
         !(has_emitting_env && settings.m_background_emits_light)))